src/go/rdctl/pkg/factoryreset/delete_data.go

```diff
@@ -65,7 +65,7 @@ func appHomeDirectories(appPaths paths.Paths) []string {
 	pathList := make([]string, 0, len(appHomeFiles))
 	for _, file := range appHomeFiles {
 		fullname := strings.ToLower(filepath.Join(appPaths.AppHome, file.Name()))
-		if !excludeDir[fullname] {
+		if excludeDir[fullname] == "" {
```

Comment:
Alternatively (and possibly clearer), `if _, ok := excludeDir[fullname]; !ok`

Reply:
I thought about it, but I actually found the current version simpler. The `ok` check is really only needed when `map` entries can legitimately be the null value of the type.